kratos/tests/cpp_tests/utilities/test_auxiliar_model_part_utilities.cpp

```diff
@@ -18,7 +18,7 @@
 #include "containers/model.h"
 #include "testing/testing.h"
 #include "includes/model_part.h"
-#include "utilities/auxiliar_model_part_utilities.h"
+#include "utilities/auxiliary_model_part_utilities.h"
```

Comment:
This filename should be changed as well.

Reply:
The name is changed

Reply:
Ahh, the test name...

Reply:
Done